medusa/black_and_white_list.py

```diff
@@ -33,22 +33,22 @@
 class BlackAndWhiteList(object):
     """Black and White List."""
 
-    def __init__(self, show):
+    def __init__(self, series):
         """Init method.
 
         :param show_id:
         :type show_id: int
         """
-        if not show.indexerid:
+        if not series.indexerid:
```

Comment:
@medariox @fernandog you might not have known this. But whenever passing a series object, i've used the variable name `series_obj` everywhere. Only when accessing the series as an attribute of for example a SearchResult class, i used `series`.